ldes-server-ingest/ldes-server-ingest-kafka/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/ingest/kafka/model/KafkaConsumerResponse.java

```diff
@@ -0,0 +1,40 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.ingest.kafka.model;
+
+import java.util.List;
+
+public record KafkaConsumerResponse(String listenerId, String groupId, Boolean active, List<KafkaConsumerAssignment> assignments) {
+	public static Builder builder() {
+		return new Builder();
+	}
```

Comment:
If you provide this static method, then I would personally make the constructor of the Builder class private

Reply:
with records, that's not possible ;)